From a8599037a1f6b69a43c7cbe034d7de613d1dfaab Mon Sep 17 00:00:00 2001
From: Zain Wang <wzz@rock-chips.com>
Date: Sat, 29 Sep 2018 10:24:40 +0800
Subject: [PATCH] arm64: dts: rockchip: update rk3328-evb bootargs

Tried to correct the right things for buildroot:

* remove rockchip_jtag
* keep console as FIQ0
* add "root=" boot option
* "earlyprintk=uart8250-32bit" -> "earlycon=uart8250,mmio32"

Change-Id: I82fa22ca7e2198dbfacbbf3ac6e951a571c3811c
Signed-off-by: Zain Wang <wzz@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3328-evb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-evb.dts b/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
index 3c8a5cc87322..ab0590888872 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
@@ -49,7 +49,7 @@
 	compatible = "rockchip,rk3328-evb", "rockchip,rk3328";
 
 	chosen {
-		bootargs = "rockchip_jtag earlyprintk=uart8250-32bit,0xff130000 swiotlb=1 kpti=0";
+		bootargs = "earlycon=uart8250,mmio32,0xff130000 swiotlb=1 kpti=0 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootwait";
 	};
 
 	fiq-debugger {
-- 
2.35.3

